### DISCUSSION TITLE
Expansion of pm.math to include more numpy functions

### DISCUSSION THREAD

**Discussion by brandonhorsley** [pymc-devs/pymc3#7124] [Ideas]
Hello all!

I raised a point on the pymc discourse (https://discourse.pymc.io/t/no-inverse-trig-in-pm-math-functions/13681/2) that pm.math appeared to be missing inverse trig functions at which point it was noted by @jessegrabowski that this is part of a larger point that pm.math is missing a lot of numpy functions and it would be worth raising this as an issue on github. As per the community guidance on contributing I am raising it here first before raising it as a formal issue.

The current workaround is to use pytensor.tensor for a range of numpy equivalent backends so this isn't a high priority point but I imagine will likely be very valuable for strengthening the documentation of pymc and making it easier for newer users by then only needing to refer to the pm.math documentation.

**Comment by jessegrabowski** [comment]
Hey Brandon!

I think you could copy/paste your message as an issue with the "Enhancement" category. It will get more visibility that way, and it can be referenced if someone takes it up. 

If you were interested in tackling this, basically all you need to do is import the desired functions in `pm.math` and add them to `__all__`. See [here](https://github.com/pymc-devs/pymc/blob/main/pymc/math.py).

**Reply by brandonhorsley** [reply]
Hi Jesse, I have opened it as an issue with the feature request label (since I am not a contributor and thus can't attach the enhancement label, also butchered the markdown on the issue lol but the point is still clear). I took a quick gloss over and noticed the operations like arccos are imported in the file you linked and included in __all__ but alas trying to use them in code like so:
<img width="620" alt="Screenshot 2024-02-03 010814" src="https://github.com/pymc-devs/pymc/assets/56071064/446bb5a3-8d8a-44d5-8bc1-d532a86dd6bc">
doesn't work, which matches with its absence in the documentation. Could this be a bug or am I missing something?
